tools/generator/form-sections.js

```diff
@@ -175,25 +175,13 @@ export function renderExperienceFragment(form, handleInput) {
   `;
 }
 
-export function renderAssetDelivery(form, handleInput) {
+export function renderAssetDelivery(form, uploadFile) {
   return html`
     <div class="form-row">
       <h2>Asset Delivery</h2>
-      ${form.contentType.includes('video') ? html`
-        <sl-input type="text" name="videoAsset" .value=${form.videoAsset} placeholder="https://video.tv.adobe.com/v/..." label="Video Asset" @input=${handleInput}></sl-input>`
-    : html`<sl-input type="file" name="pdfAsset" label="Upload PDF Asset" @input=${handleInput}></sl-input>`}
-    </div>
-  `;
-}
-
-export function renderUrl(form, handleInput) {
-  const pathPlaceholder = `/resources/${form.marqueeHeadline ? form.marqueeHeadline.toLowerCase().replace(/\s+/g, '-') : ''}.html`;
-  form.url = pathPlaceholder;
-
-  return html`
-    <div class="form-row">
-      <h2>URL</h2>
-      <sl-input type="text" name="url" .value=${form.url} placeholder="${form.marqueeHeadline ? pathPlaceholder : ''}" label="URL" @input=${handleInput}></sl-input>
+      ${(form.contentType || '').toLowerCase().includes('video') ? html`
```

Comment:
I don't usually see this pattern. Why not do
```suggestion
      ${form.contentType?.toLowerCase().includes('video') ? html`
```